packages/react-dom/src/events/AuxClickEventPlugin.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import {accumulateTwoPhaseDispatches} from 'events/EventPropagators';
+
+import SyntheticMouseEvent from './SyntheticMouseEvent';
+
+const eventTypes = {
+  auxClick: {
+    phasedRegistrationNames: {
+      bubbled: 'onAuxClick',
+      captured: 'onAuxClickCapture',
+    },
+    dependencies: ['topAuxClick', 'topClick'],
+  },
+};
+
+const AuxClickEventPlugin = {
+  eventTypes,
+
+  extractEvents(
+    topLevelType: mixed,
+    targetInst: mixed,
+    nativeEvent: MouseEvent,
+    nativeEventTarget: EventTarget,
+  ) {
+    if (topLevelType === 'topClick' && nativeEvent.button === 0) {
```

Comment:
When is button not 0? Does this account for right clicks? Is that even a problem?

Reply:
this is any click that isn't a left click yeah. which is correct going off the mdn example https://mdn.github.io/dom-examples/auxclick/